securedrop/tests/test_template_filters.py

```diff
@@ -19,10 +19,7 @@
 class TestTemplateFilters(object):
 
     def get_fake_config(self):
-        class Config:
-            def __getattr__(self, name):
-                return getattr(config, name)
-        return Config()
+        return SDConfig()
```

Comment:
thanks to the new **SDConfig** there is no need for **get_fake_config(self)** and all calls to **get_fake_config()** can be replaced with **SDConfig()** instead :+1:

Reply:
I will update the tests in a new PR where will change how we deal with configuration. This PR is keeping things in API level same, and we should not update any tests, that will confirm that our old will work in the same way with the new `SDConfig`.

Reply:
ok